foursight_core/react/api/yaml_utils.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+
+
+def load_yaml(yaml_string: str) -> dict:
+    """
+    Loads the given YAML string and returns it as a dictionary.
+    If an error occurs on load then and empty dictionary is returned.
+    """
+    try:
+        return yaml.load(yaml_string, Loader=yaml.Loader)
+    except Exception:
+        return {"error": "Cannot parse YAML."}
```

Comment:
Using `get_error_message` from `dcicutils.misc_utils` you might want this (untested):

```suggestion
    except Exception as e:
        return {"error": f"Cannot parse YAML: {get_error_message(e)}"}
```